solutionbox/inception/release.sh

```diff
@@ -14,6 +14,8 @@
 
 # Build a distribution package
 python setup.py sdist --formats=gztar
+
+# Release to gs
 gsutil cp ./dist/inception-0.1.tar.gz gs://cloud-datalab/packages/inception-0.1.tar.gz
```

Comment:
The file is there doesn't mean everyone can run it. :) But it is good to have it so you probably can release a new version too conveniently.